src/frontend/js/components/RegisteredAddress/index.tsx

```diff
@@ -70,7 +70,12 @@ const RegisteredAddress = ({ promote, select, edit, remove, address }: Props) =>
           }`}
         />
       </button>
-      <strong className="h6 registered-addresses-item__title">{address.title}</strong>
+      <strong
+        data-testid={`address-${address.id}-title`}
+        className="h6 registered-addresses-item__title"
+      >
+        {address.title}
+      </strong>
```

Comment:
Are those changes in this file really related to this commit ?

Reply:
Not really, it's a workaround to prevent test failure when address factory generates several addresses with the same title. As this concern tests, I embedded in this commit but if you prefer I can create a separated commit for this.

Reply:
I'm ok with this as long as it is intended. :+1: